docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -161,43 +161,6 @@ If you have include files in non-standard places you can pass an
         ext_modules=cythonize("src/*.pyx", include_path=[...]),
     )
 
-Often, Python packages that offer a C-level API provide a way to find
```

Comment:
I wonder if it's worth leaving one sentence here with a link. Just something like: "More details on building Cython modules that use `cimport numpy` can be found in the Numpy section of the user guide"

Reply:
I put it right to the beginning of the chapter because I suppose that the user looking for quick way of compilation with numpy does not care about other details...